test/method_not_allow.test.js

```diff
@@ -59,11 +59,4 @@ describe('test/method_not_allow.test.js', () => {
       .set('accept', 'text/html')
       .expect(405);
   });
-
-  it('should not allow option method', () => {
```

Comment:
keep this test and make sure we support option method.

Reply:
Fixed but change to `should allow`.